mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -324,7 +324,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_5670 baseTick ()V
 	METHOD method_5671 getCommandSource ()Lnet/minecraft/class_2168;
 	METHOD method_5672 getHighSpeedSplashSound ()Lnet/minecraft/class_3414;
-	METHOD method_5673 equipStack (Lnet/minecraft/class_1304;Lnet/minecraft/class_1799;)V
+	METHOD method_5673 setStack (Lnet/minecraft/class_1304;Lnet/minecraft/class_1799;)V
```

Comment:
This method has some logic in it, it's not a simple setter. It's even a no-op in `Entity`, which is very counter-intuitive for a setter (setters that are not supported by a class would generally throw an `UnsupportedOperationException`).

Reply:
Ther no-op method in Entity is just bad Mojang design from what I can tell, and shouldn't be considered when naming the methods. As for the "side-effects", all there is as far as that is concerned is a *callback* method. The callback is what has side-effects, not the setter itself. Your logic could also argue that `World#setBlockState` is not a setter because it also has side-effects.

Reply:
Sure, I would rather have a non-setter name for `setBlockState`. I don't have a suggestion though, so I don't complain about it. However `equipStack` is already a good name in my opinion, and I consider changing it to a nondescript setter a regression.

Reply:
I would expect a method such as `equipStack` to return the previously equipped stack, but this method returns void like a typical setter.

Reply:
It's also worth noting that the "side-effects" only exist for 2 entity types, player and armor stand. For every other living entity, it is merely a standard setter.

Reply:
I was more thinking of the side effect in `AbstractSkeletonEntity`, which causes it to update its AI.

Reply:
Again, that is merely a callback for when the stack changes as the AI needs to update how it handles the equipped item if it differs from the previously equipped item.